pkg/kine/logstructured/sqllog/sql.go

```diff
@@ -124,17 +124,25 @@ func (s *SQLLog) DoCompact(ctx context.Context) error {
 }
 
 func (s *SQLLog) compactor(ctx context.Context, nextEnd int64) (int64, error) {
+	_, span := tracer.Start(ctx, "sqllog.compactor")
```

Comment:
I think we should use the returned context, so that if we create a span in `CurrentRevision` and `GetCompactRevision` (or any other method) it will be grouped under `sqllog.compactor`.